src/plugins/package/azurePackage.ts

```diff
@@ -1,54 +1,44 @@
 
 import Serverless from "serverless";
 import AzureProvider from "../../provider/azureProvider";
-import { BindingUtils } from "../../shared/bindings";
-import { Utils } from "../../shared/utils";
-import fs from "fs";
-import path from "path";
+import { PackageService } from "../../services/packageService";
 
 export class AzurePackage {
+  private bindingsCreated: boolean = false;
+  private packageService: PackageService;
   public provider: AzureProvider;
   public hooks: { [eventName: string]: Promise<any> };
 
-  public constructor(private serverless: Serverless, private options: Serverless.Options) {
+  public constructor(private serverless: Serverless) {
```

Comment:
Reason for no options?

Reply:
The plugin doesn't use any options right now - so removed it.  If it starts to take options in the future I'm fine bringing it back.